src/main/java/com/iexec/common/contract/generated/App.java

```diff
@@ -20,6 +20,8 @@
 import java.util.Arrays;
```

Comment:
You should not update what's inside /generated

Reply:
If you have time you can setup sonar exclusion for such folder https://docs.sonarqube.org/latest/project-administration/narrowing-the-focus/

Reply:
Those are reverted.
That'd be great to set Sonar exclusion but we still can't sync our local Sonar settings with SonarQube so it would still annoy me :laughing:

Reply:
It should be OK in SonarQube:
![image](https://user-images.githubusercontent.com/87655102/149915832-6bc039d9-6569-4f29-9a45-e26f1d665d58.png)